nltk/metrics/index.html

```diff
@@ -0,0 +1,629 @@
+
```

Comment:
@53X is this html file mistakenly added to the git push? Could you got remove this html file?

Reply:
Removed